packages/design-system/src/components/Tooltip/Tooltip.tsx

```diff
@@ -107,7 +107,7 @@ export interface TooltipProps {
  */
 export const Tooltip = (props: TooltipProps) => {
   const popper = useRef(null);
-  const id = useId('tooltip-trigger--', props.id);
+  const contentId = useId('tooltip-trigger--', props.id);
```

Comment:
renaming from `id` to `contentId` fixed some confusion happening in the trigger. the trigger was trying to apply the `props.id` value to `aria-describedby` instead of the value saved in this `id` const.